lib/assert.js

```diff
@@ -931,6 +933,87 @@ assert.doesNotMatch = function doesNotMatch(string, regexp, message) {
   internalMatch(string, regexp, message, doesNotMatch);
 };
 
+const noop = () => {};
+
+class Tracker {
+
+  constructor() {
+    this.callChecks = new SafeSet();
+  }
+
+  calls(fn, exact = 1) {
+    if (process._exiting)
+      throw new ERR_CALLS_UNAVAILABLE_DURING_EXIT();
```

Comment:
This error code would likely need to be changed as it's not happening during exit anymore.

Reply:
@mcollina ... not sure what you mean... the error is only thrown while `process._exiting` is true.

Reply:
My bad, forget about this then!